apps/android/app/src/main/java/chat/simplex/app/model/SimpleXAPI.kt

```diff
@@ -183,7 +192,8 @@ open class ChatController(private val ctrl: ChatCtrl, val ntfManager: NtfManager
   private var receiverStarted = false
 
   init {
-    chatModel.runServiceInBackground.value = appPrefs.runServiceInBackground.get()
+    chatModel.notificationsMode.value = NotificationsMode.valueOf(appPrefs.notificationsMode.get()!!)
+    chatModel.notificationPreviewMode.value = NotificationPreviewMode.valueOf(appPrefs.notificationPreviewMode.get()!!)
```

Comment:
let's please catch conversion exception here and fallback to default in case it happens, instead of '!!'.